src/vector/mod.rs

```diff
@@ -22,13 +22,15 @@ mod geo_to_gdal;
 mod geometry;
 mod layer;
 mod ops;
+mod sql;
 
 pub use defn::{Defn, Field, FieldIterator};
 pub use feature::{Feature, FieldValue, FieldValueIterator};
 pub use gdal_sys::{OGRFieldType, OGRwkbGeometryType};
 pub use geometry::Geometry;
 pub use layer::{FeatureIterator, FieldDefn, Layer};
 pub use ops::GeometryIntersection;
+pub use sql::ResultSet;
```

Comment:
This is a pretty general name. Should this maybe be `use sql::ResultSet as SqlResultSet` or something to namespace it in the crate namespace?

Just a minor thought, feel free to ignore this feedback.